dependencies.yaml

```diff
@@ -639,6 +639,8 @@ dependencies:
           - pytest-benchmark
           - pytest-cases>=3.8.2
           - scipy
+          - mmh3
+          - xxhash
```

Comment:
We need to install [python-xxhash](https://anaconda.org/conda-forge/python-xxhash) in conda, and [xxhash](https://pypi.org/project/xxhash/) for requirements/pyproject. (If you use `xxhash`, you'll only get the C++ library from conda-forge.) Can you split this?

Reply:
done (I think) let me know if https://github.com/rapidsai/cudf/pull/15418/commits/642b4446d029aacdc33ca588cef61c2bd617c392 looks right to you.